mlir/lib/Dialect/Rock/Pipelines/Pipelines.cpp

```diff
@@ -204,7 +204,10 @@ void rock::buildKernelPipeline(OpPassManager &pm,
     funcPm.addPass(rock::createRockSugarToLoopsPass());
     funcPm.addPass(rock::createRockCleanMathPass());
     math::MathExtendToSupportedTypesOptions extendToLLVMTypesOptions;
-    extendToLLVMTypesOptions.extraTypeStrs = {"f16"};
+    SmallVector<std::string, 1> supportedFloats = {"f16"};
+    extendToLLVMTypesOptions.extraTypeStrs.assign(supportedFloats.begin(),
```

Comment:
Do you still require this PR  ? 

This was merged recently 

https://github.com/ROCm/rocMLIR/pull/1932/files